central/role/resources/list.go

```diff
@@ -107,6 +107,11 @@ var (
 	// To-be-deprecated in 3.75 with ROX-12750 (deprecation notice in 3.73).
 	ServiceIdentity = newDeprecatedResourceMetadata("ServiceIdentity",
 		permissions.GlobalScope, Administration)
+	SignatureIntegration = newDeprecatedResourceMetadata("SignatureIntegration",
```

Comment:
I'm curious why these are in the deprecated section?

Reply:
The feature was branched off before a lot of deprecated SAC were dropped I believe. Need to ask about this, but you're right, probably makes sense to keep it within the new hierarchy.

Reply:
Looks like I was mistaken about the meaning of this construction, the advice I've got was to use `DeploymentExtension` resource as other similar objects.